src/web/handlers/GestureHandler.ts

```diff
@@ -749,9 +749,7 @@ export default abstract class GestureHandler {
   protected resetConfig(): void {}
 
   public onDestroy(): void {
-    (this.delegate as GestureHandlerWebDelegate).removeContextMenuListeners(
-      this.config
-    );
+    (this.delegate as GestureHandlerWebDelegate).destroy(this.config);
```

Comment:
Instead of casting you can add this method to the `GestureHandlerDelegate` interface